
Showing logs for build [32mvicky-socs-different-branch-tri-8zzhh-1[0m stage [32mfrom-build-pack[0m and container [32mstep-credential-initializer-jhzdt[0m
{"level":"warn","ts":1584442267.7733595,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1584442267.7736652,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mvicky-socs-different-branch-tri-8zzhh-1[0m stage [32mfrom-build-pack[0m and container [32mstep-working-dir-initializer-6wdvq[0m
{"level":"warn","ts":1584442268.864888,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1584442268.8678844,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mvicky-socs-different-branch-tri-8zzhh-1[0m stage [32mfrom-build-pack[0m and container [32mstep-place-tools[0m

Showing logs for build [32mvicky-socs-different-branch-tri-8zzhh-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-source-vicky-socs-different-branch-tri-8zzhh-pqkph[0m
{"level":"warn","ts":1584442273.9860942,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1584442276.565989,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/vicky-socs/different-branch-trigger.git @ v0.0.1 in path /workspace/source"}

Showing logs for build [32mvicky-socs-different-branch-tri-8zzhh-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:0290c5e2057936acb487ffa84dd346684bc1499c
WARNING: no SHAs to merge, falling back to initial cloned commit

Showing logs for build [32mvicky-socs-different-branch-tri-8zzhh-1[0m stage [32mfrom-build-pack[0m and container [32mstep-setup-jx-git-credentials[0m
Generated Git credentials file /workspace/xdg_config/git/credentials

Showing logs for build [32mvicky-socs-different-branch-tri-8zzhh-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-python-unittest[0m

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK

Showing logs for build [32mvicky-socs-different-branch-tri-8zzhh-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-container-build[0m
[36mINFO[0m[0000] Downloading base image python:onbuild        
2020/03/17 10:51:20 No matching credentials were found, falling back on anonymous
[36mINFO[0m[0002] Error while retrieving image from cache: getting file info: stat /workspace/sha256:43780a8431c53f4afd734ad795e03127c0c607337a60639c5903a51a03535c22: no such file or directory 
[36mINFO[0m[0002] Downloading base image python:onbuild        
2020/03/17 10:51:22 No matching credentials were found, falling back on anonymous
[36mINFO[0m[0004] Executing 3 build triggers                   
[36mINFO[0m[0004] Using files from context: [/workspace/source/requirements.txt] 
[36mINFO[0m[0004] Checking for cached layer 685249416972.dkr.ecr.ap-south-1.amazonaws.com/todo/cache:20ed5631d983ab35ddcc4224cde40d34ba4dcc68c4ee26635bbf30d01c3c2af6... 
[36mINFO[0m[0004] Using caching version of cmd: RUN pip install --no-cache-dir -r requirements.txt 
[36mINFO[0m[0004] Using files from context: [/workspace/source] 
[36mINFO[0m[0004] Using files from context: [/workspace/source/requirements.txt] 
[36mINFO[0m[0004] cmd: EXPOSE                                  
[36mINFO[0m[0004] Adding exposed port: 8080/tcp                
[36mINFO[0m[0004] Skipping unpacking as no commands require it. 
[36mINFO[0m[0004] Taking snapshot of full filesystem...        
[36mINFO[0m[0004] Using files from context: [/workspace/source/requirements.txt] 
[36mINFO[0m[0004] COPY requirements.txt /usr/src/app/          
[36mINFO[0m[0004] Taking snapshot of files...                  
[36mINFO[0m[0004] RUN pip install --no-cache-dir -r requirements.txt 
[36mINFO[0m[0004] Found cached layer, extracting to filesystem 
[36mINFO[0m[0004] Taking snapshot of files...                  
[36mINFO[0m[0005] Using files from context: [/workspace/source] 
[36mINFO[0m[0005] COPY . /usr/src/app                          
[36mINFO[0m[0005] Taking snapshot of files...                  
[36mINFO[0m[0005] Using files from context: [/workspace/source/requirements.txt] 
[36mINFO[0m[0005] COPY requirements.txt .                      
[36mINFO[0m[0005] Taking snapshot of files...                  
[36mINFO[0m[0005] ENV PORT 8080                                
[36mINFO[0m[0005] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0005] EXPOSE 8080                                  
[36mINFO[0m[0005] cmd: EXPOSE                                  
[36mINFO[0m[0005] Adding exposed port: 8080/tcp                
[36mINFO[0m[0005] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0005] ENTRYPOINT ["python"]                        
[36mINFO[0m[0005] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0005] CMD ["app.py"]                               
[36mINFO[0m[0005] No files changed in this command, skipping snapshotting. 
2020/03/17 10:51:25 pushed blob: sha256:a9abf7c48711922911c040c9ad94698f5d5bafe737405d095ca9908b55baa3e1
2020/03/17 10:51:25 pushed blob: sha256:eea3fab059ddcb1756f50f6009d4cc04738993306cfe84ad57a76aa381250877
2020/03/17 10:51:25 pushed blob: sha256:cba502be25797320ca5f5d74257f5b3cabfe84dc92e6bf5785a7987cbfff5746
2020/03/17 10:51:26 pushed blob: sha256:6bfca92305ea2f1ad12e440377a46c4d299086cc0653521692c659aa36d0e703
2020/03/17 10:51:26 pushed blob: sha256:687b097229940887d9134885dee0f4cc48759753bcd1788bb89f714f121ebe51
2020/03/17 10:51:26 pushed blob: sha256:490280dbcfb772eb5731f059e38fcfacaeb75b4d005af7c249fd51926db2bd62
2020/03/17 10:51:26 pushed blob: sha256:e2c0fb34dff3a1d32346b92e53fb88a14d1987b402a2348cb379c039687fabe2
2020/03/17 10:51:27 pushed blob: sha256:e96bc319bce0351d1157092352bc5f85f8800c745388e60e86d06dec99285b45
2020/03/17 10:51:27 pushed blob: sha256:2d21c767035c4593de8bf7ad9781aa83ba110871d1a50871192f4e3480e6f11d
2020/03/17 10:51:27 pushed blob: sha256:ece825d3308bbf98f708d95475ccd880497a1e2979ff4ef2534d49049a18488f
2020/03/17 10:51:28 pushed blob: sha256:e10b68fb77e6e91453cd4c1b02e2d77aa23a857cd8eb250617129d074478e1a5
2020/03/17 10:51:28 pushed blob: sha256:06854774e2f35e4763103a446112c67322913a6177bea69eb49663e0166e6d6d
2020/03/17 10:51:31 pushed blob: sha256:1c7fe136a31e13a1c879245672184ced3b9fc590989ba42a522a57b071a9c1aa
2020/03/17 10:51:34 pushed blob: sha256:f0db43b9b8da254cebc77bf871006df01219e195b9853616b9f708f124a00f3a
2020/03/17 10:51:34 685249416972.dkr.ecr.ap-south-1.amazonaws.com/vicky-socs/different-branch-trigger:0.0.1: digest: sha256:7bf84e1c6bd2d2ccce7137fff15ed8949733a10a72cc64540ccf9bb57e912fee size: 2385

Showing logs for build [32mvicky-socs-different-branch-tri-8zzhh-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-post-build[0m
no CVE provider running in the current ** namespace so skip adding image to be analysed

Showing logs for build [32mvicky-socs-different-branch-tri-8zzhh-1[0m stage [32mfrom-build-pack[0m and container [32mstep-promote-changelog[0m
/bin/sh: line 0: cd: /workspace/source/charts/different-branch-trigger: No such file or directory
[31m
Pipeline failed on stage 'from-build-pack' : container 'step-promote-changelog'. The execution of the pipeline has stopped.[0m
